tests/unit/test_tools.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import pytest
+
+from rocketpy.tools import euler_to_quaternions
+
+
+def test_euler_to_quaternions():
+    q0, q1, q2, q3 = euler_to_quaternions(0, 0, 0)
+    assert q0 == 1
+    assert q1 == 0
+    assert q2 == 0
+    assert q3 == 0
+    q0, q1, q2, q3 = euler_to_quaternions(90, 90, 90)
+    assert round(q0, 7) == 0.7071068
+    assert round(q1, 7) == 0
+    assert round(q2, 7) == 0.7071068
+    assert round(q3, 7) == 0
```

Comment:
Using `@pytest.mark.parametrize` makes the code more scalable (i.e. add more cases in the future) and allow us to run one case without running the other.

```suggestion
@pytest.mark.parametrize("angles, expected_quaternions", [
    ((0, 0, 0), (1, 0, 0, 0)),
    ((90, 90, 90), (0.7071068, 0, 0.7071068, 0))
])
def test_euler_to_quaternions(angles, expected_quaternions):
    q0, q1, q2, q3 = euler_to_quaternions(*angles)
    assert round(q0, 7) == expected_quaternions[0]
    assert round(q1, 7) == expected_quaternions[1]
    assert round(q2, 7) == expected_quaternions[2]
    assert round(q3, 7) == expected_quaternions[3]
```